src/fosslight_prechecker/cli.py

```
@@ -61,16 +61,33 @@ def main():
+                if 'licenses' in str(f).lower() and f.locate().is_file()
+            ]
+        for fpath in sorted(license_files):
+            try:
+                with open(fpath, 'r', encoding='utf-8') as f:
+                    content = f.read()
+                try:
+                    print(content)
+                except UnicodeEncodeError:
+                    sys.stdout.buffer.write(content.encode('utf-8', errors='replace') + b'\n')
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)